yarn.lock

```diff
@@ -2991,6 +2991,13 @@
   resolved "https://registry.yarnpkg.com/@types/vscode/-/vscode-1.85.0.tgz#46beb07f0f626665b52d1e2294382b2bc63b602e"
   integrity sha512-CF/RBon/GXwdfmnjZj0WTUMZN5H6YITOfBCP4iEZlOtVQXuzw6t7Le7+cR+7JzdMrnlm7Mfp49Oj2TuSXIWo3g==
 
+"@types/ws@^8.5.10":
```

Comment:
Please replace all `https://npmjs-registry.ivyteam.io` with `https://registry.yarnpkg.com` in this file. Thanks.